include/keyence/impl/high_speed_defs.h

```diff
@@ -3,9 +3,28 @@
 
 #include <stdint.h>
 
+#define KEYENCE_FUNDAMENTAL_LENGTH_UNIT (0.01 * 1.0e-6)
+// The integer encoded values representing 'out of range'
+// received from the sensor
+#define KEYENCE_INFINITE_DISTANCE_VALUE -524288
```

Comment:
There are really two values that is returns?  How did you determine these values?  Wouldn't all *bad* values be negative or something more logical?